src/main/java/pokecube/core/moves/animations/MoveAnimationBase.java

```diff
@@ -99,7 +99,7 @@ public void reallyInitRGBA()
             catch (final Exception e1)
             {
                 for (final DyeColor col : DyeColor.values())
-                    if (col.getName().equals(val))
+                    if (col.name().equals(val))
```

Comment:
in this case, it should use "toString()" or "getString()" (probably this one) rather than "name"